lib/ReactViews/Analytics/PolygonParameterEditor.jsx

```diff
@@ -38,7 +38,7 @@ const PolygonParameterEditor = React.createClass({
     },
 
     getValue() {
-        const pointsLongLats = this.props.parameterValues[this.props.parameter.id];
+        const pointsLongLats = this.props.parameterValues[this.props.parameter.id][0];
```

Comment:
@RacingTadpole good observation. Yeah, if you could log that as a separate issue that would be great. Thanks!